projects/explorer/README.md

```diff
@@ -0,0 +1,64 @@
+# Explorer
+
+## Directory
+
+[https://github.com/UnUniFi/web-apps/tree/main/projects/explorer](https://github.com/UnUniFi/web-apps/tree/main/projects/explorer)
+
+## Environments
+
+| environment | url | branch |
+| - | - | - |
+| alpha-test | [https://alpha-test.ununifi.io/explorer/](https://alpha-test.ununifi.io/explorer/) | develop(latest) |
+| beta-test | [https://beta-test.ununifi.io/explorer/](https://beta-test.ununifi.io/explorer/) | release(latest) |
+| test | in preparation | main(latest) |
+| main(beta) | [https://ununifi.io/explorer/](https://ununifi.io/explorer/) | main |
+
+## Setup
+
+### Install Node.js
```

Comment:
need node version

Reply:
Currently, I think it works on both 14.x or 16.x. (Currently, I have debug with 16.x.)
But, you mean it will not work on 16.x with @cosmos-client/core compatible to Cosmos SDK v0.46.x ?
I think it should be noted after @cosmos-client/core version up.
How do you think about it?

Reply:
Oh, but, I didn't check 18.x ...
OK. I will write to recommend 14.x.
Thanks.

Reply:
ok

Reply:
I have added recommendation comment for Node.js v14.